pybossa/view/projects.py

```diff
@@ -3548,6 +3548,7 @@ def integrate_ext_config(config_dict):
             if bool(data_access_levels):
                 # for private gigwork
                 project.info['data_access'] = data.get('data_access')
+            project.info['completed_tasks_cleanup_days'] = data.get('completed_tasks_cleanup_days')
```

Comment:
As its optional, can we not add the property when it doesn't exist under data payload? Also ensure its one of the permitted values only
```python
PERMITTED_CLEANUP_DAYS = [30, 60, 180]
completed_tasks_cleanup_days = int(data.get('completed_tasks_cleanup_days', 0)) # confirm this doesn't assert with non int values
if completed_tasks_cleanup_days in PERMITTED_CLEANUP_DAYS:
       project.info['completed_tasks_cleanup_days'] =  completed_tasks_cleanup_days
```

Reply:
With backend validation is ideal. But I don't think we need here as this API is only callable from UI

Reply:
Sounds good. Thanks for confirming.